Foundation/testsuite/src/TestApp.cpp

```diff
@@ -57,6 +60,32 @@ int main(int argc, char** argv)
 				std::cout << argv[i] << std::endl;
 			}
 		}
+		else if (arg.find("--pidfile") != std::string::npos || arg.find("-p") != std::string::npos)
+		{
+			std::unique_ptr<PIDFile> _pPIDFile;
+			size_t equals_pos = arg.find('=');
+
+			if (equals_pos != std::string::npos)
+			{
+				std::string pidPath = arg.substr(equals_pos + 1);
+				_pPIDFile.reset(new PIDFile(pidPath, true, false));
+				_pPIDFile->create(false);
+
+				sigset_t sset;
```

Comment:
this is POSIX-only, need at least windows implementation, too. #ifdef out the complete `ProcessRunner` code out for android (@zaleksandar should try at least android build/testrun before relase to make sure we did not break something there)

Reply:
Implemented in https://github.com/pocoproject/poco/pull/4225/commits/731d77093525dc9db472cd73b9b66d3c1ade11ec